styles.css

```diff
@@ -70,3 +70,126 @@ a {
 ul {
   padding-inline-start: 0;
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+}
+
+.port1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  max-width: 327px;
+  margin: auto;
+  margin-bottom: 70px;
+}
+
+.title {
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.clnt {
+  font-weight: 600;
+  font-size: 13px;
+  color: #344563;
+}
+
+.deets {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  color: #7a869a;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 205px;
+  padding: 0;
+  list-style: none;
+  gap: 8px;
+}
+
+.tgs {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.prjctbtn {
+  display: flex;
+  align-items: center;
+  background: none;
+  border: 2px solid #396df2;
+  border-radius: 10px;
+  padding: 8px 16px;
+}
+
+.prjctbtn:hover {
+  background: #396df2;
+}
```

Comment:
The `:hover` is a very useful and quite simple animation 🤹🏽  but to make yours better you have to also make sure that the text color is also changed when the button is hovered. Silly mistake 😅 I made it once or twice too. Your code could become this:
```
.prjctbtn:hover {
  background: #396df2;
  color: white;
}
```